app/views/users/user_tweets.erb

```diff
@@ -0,0 +1,5 @@
+<h2>These tweets were written by <%=@user.username%>!</h2>
+<% @user.tweets.each do |tweet| %>
```

Comment:
You should avoid database calls on the front end. We should set `@user_tweets = @user.tweets` in the TweetsController